src/test/java/gov/nih/nci/evs/api/controller/HgncSampleTest.java

```diff
@@ -69,18 +62,26 @@ public void testHGNCTerminology() throws Exception {
 
     url = "/api/v1/metadata/terminologies";
     log.info("Testing url - " + url);
-    result = testMvc.perform(get(url).param("latest", "true").param("terminology", "hgnc")).andExpect(status().isOk())
-        .andReturn();
+    result =
+        testMvc
+            .perform(get(url).param("latest", "true").param("terminology", "hgnc"))
+            .andExpect(status().isOk())
+            .andReturn();
     content = result.getResponse().getContentAsString();
     log.info(" content = " + content);
 
     final List<Terminology> terminologies =
-        new ObjectMapper().readValue(content, new TypeReference<List<Terminology>>() {
-          // n/a
-        });
+        new ObjectMapper()
```

Comment:
The line breaks seem rather aggressive, to the point of affecting readability. Did we change our formatting recently?

Reply:
This was formatted using the google-java-format plugin.